core/ui/PageControls/new-journal.tid

```diff
@@ -16,5 +17,7 @@ description: {{$:/language/Buttons/NewJournal/Hint}}
 \end
 <$set name="journalTitleTemplate" value={{$:/config/NewJournal/Title}}>
 <$set name="journalTags" value={{$:/config/NewJournal/Tags}}>
+<$set name="journalText" value={{$:/config/NewJournal/Text}}>
```

Comment:
Where is the content of such field configured? I miss an interface on the control panel,where you can set the tags and the title.